app/containers/MessageBox/index.tsx

```diff
@@ -1090,7 +1090,6 @@ class MessageBox extends Component<IMessageBoxProps, IMessageBoxState> {
 				<TextInput
 					ref={component => (this.component = component)}
 					style={[styles.textBoxInput, { color: themes[theme].bodyText }]}
-					// @ts-ignore
```

Comment:
For some reason this raises an error like if the type is wrong, even when the type you're sending is either common or platform-specific. Might be related to our `react-native` version, quite unsure honestly.